app/components/data-download.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { RDFCubeViewQueryMock } from "@/test/cube-view-query-mock";
+RDFCubeViewQueryMock;
+
+// eslint-disable-next-line import/order
+import get from "lodash/get";
```

Comment:
For my information, why do we need to disable import/order here ?

Reply:
That's a good question, it had to do with the order of mocking things – in some tests, this mock had to be the first thing that's mocked, which clashes with the default import order we define via eslint. In fact, I believe it makes more sense to include this mock in jest setup – I adjusted this in https://github.com/visualize-admin/visualization-tool/pull/1420/commits/7dfce84e672dab5e816e434037d797e98581b200 👀

Reply:
Looks good, thanks 👍